edb/lib/std/30-datetimefuncs.edgeql

```diff
@@ -56,32 +56,33 @@ std::datetime_get(dt: std::datetime, el: std::str) -> std::float64
 
 
 CREATE FUNCTION
-std::duration_get(dt: std::duration, el: std::str) -> std::float64
+std::datetime_trunc(dt: std::datetime, unit: std::str) -> std::datetime
 {
-    SET volatility := 'IMMUTABLE';
+    # date_trunc of timestamptz is STABLE in PostgreSQL
+    SET volatility := 'STABLE';
     USING SQL $$
-    SELECT date_part("el", "dt")
+    SELECT date_trunc("unit", "dt")
     $$;
 };
 
 
 CREATE FUNCTION
-std::datetime_trunc(dt: std::datetime, unit: std::str) -> std::datetime
+std::duration_trunc(dt: std::duration, unit: std::str) -> std::duration
```

Comment:
If `duration` is absolute time expressed unambiguously then `duration_trunc` is nonsensical for any unit above 'hours'. In fact, it should not even acknowledge units bigger than 'hours`. It may be superfluous. Why do we need it?

This function is more useful for `relativedelda` in my opinion.